library/core/tests/num/int_macros.rs

```diff
@@ -290,38 +290,6 @@ macro_rules! int_module {
                 assert_eq!(r.saturating_pow(0), 1 as $T);
             }
 
-            #[test]
-            fn test_isqrt() {
-                assert_eq!($T::MIN.checked_isqrt(), None);
-                assert_eq!((-1 as $T).checked_isqrt(), None);
-                assert_eq!((0 as $T).isqrt(), 0 as $T);
-                assert_eq!((1 as $T).isqrt(), 1 as $T);
-                assert_eq!((2 as $T).isqrt(), 1 as $T);
-                assert_eq!((99 as $T).isqrt(), 9 as $T);
-                assert_eq!((100 as $T).isqrt(), 10 as $T);
-            }
```

Comment:
You can probably read this test as a quick check that is easy to read, maybe just with a comment saying more tests are in `int_sqrt`.

Reply:
I was copying the setup for `ilog10`. It contains no tests in `library/core/tests/num/int_macros.rs`, including them in its own file at `library/core/tests/num/int_log.rs`.